pages/dkp/kommander/2.2/legal/version-policy/index.md

```diff
@@ -5,6 +5,7 @@ title: Version Support Policy
 menuWeight: 5
 excerpt: Supported version policy for Kommander
 beta: false
+tech-preview: true
```

Comment:
@nataliepina did I apply it right?

Reply:
Yeah, that's how i'd assume it would work, similar to the rest. I wasn't sure about the dash vs camelCase.

Reply:
Update: it is now `techPreview: true`